app.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+const path = require('path');
+
+const bodyParser = require('body-parser');
+const express = require('express');
+
+const config = require('./app/config/').Config;
```

Comment:
I was planning to add a bunch of files in config dir (like headers), so it definitely is not export single file.
That is not about to shorten the path but more about the encapsulation of the package I would say.
Anyway, I'll stick to requiring with whole path